netci.groovy

```diff
@@ -100,12 +102,13 @@ branchList.each { branchName ->
 }
 
 // **************************
-// Define outerloop windows testing.  Run locally on each machine.
+// Define outerloop windows/ubuntu testing.  Run locally on each machine.
 // **************************
 
-def osShortName = ['Windows 10': 'win10', 'Windows 7' : 'win7', 'Windows_NT' : 'windows_nt']
+def osShortName = ['Windows 10': 'win10', 'Windows 7' : 'win7', 'Windows_NT' : 'windows_nt', 'Ubuntu' : 'ubuntu']
+def outerloopOSs = ['Windows 10', 'Windows 7', 'Windows_NT', 'Ubuntu']
```

Comment:
Can you change this to Ubuntu14.04 to be more specific (since we have 15.10 now too)